src/controllers/organization.ts

```diff
@@ -106,22 +109,42 @@ export const postJoinOrganizationMiddleware = async (
     });
 
     const { confirmed, siret } = await schema.parseAsync(req.body);
+    const { id: user_id } = getUserFromAuthenticatedSession(req);
+
+    if (req.session.certificationDirigeantRequested) {
+      const isExecutive = await isOrganizationExecutive(siret, user_id);
+      if (!isExecutive) throw new InvalidCertificationError();
```

Comment:
Thoughts: I'm not sure this exception should be thrown here. I was expecting this check to be handled in the interaction/end controller, alongside the other ACR checks.